apps/api-testing/tests/subscriptions/subscriptions-get.test.js

```diff
@@ -0,0 +1,49 @@
+// @ts-nocheck
+import { expect } from 'chai';
+import { validateSchema } from '../../utils/schema-validation.js';
+import { endpoint, request, getResponse400, getResponse404 } from './index.js';
+
+describe(`GET ${endpoint}`, () => {
+	beforeEach(() => {
+		request.clear();
+	});
+
+	describe('Negative', () => {
+		it('should return a 400 status code and error response for missing required parameters', async () => {
+			const { body, statusCode } = await request.get({});
```

Comment:
You don't need to refactor these, but where tests all look similar it can be clearer to have an array of tests and use [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout), e.g. https://github.com/Planning-Inspectorate/back-office/blob/main/apps/api/src/server/applications/subscriptions/__tests__/subscriptions.test.js#L73

Reply:
Noted. Since the framework is still developing, we can adopt this approach moving forward.